spec/comment_spec.rb

```diff
@@ -0,0 +1,47 @@
+require_relative './rails_helper'
+
+RSpec.describe Comment, type: :model do
```

Comment:
- [ ] Some of your tests are failing. Kindly fix them.

  ![image](https://github.com/djo1975/Bloger/assets/81468675/18fc40f5-2deb-4556-84f4-fcf11f2562cb)

- [ ] **[OPTIONAL]** I suggest you move this model test file into a `spec/models` folder for better code organization.

Reply:
djo@djo-HP-Pavilion-15-Notebook-PC:~/Desktop/Blog/bloger$ bundle exec rspec
..........

Finished in 0.67125 seconds (files took 4.63 seconds to load)
10 examples, 0 failures